CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to Contribute to LoT
+
+We'd love your help!
+
+LoT is [Apache License 2.0](LICENSE) and accepts contributions via
+GitHub pull requests. This document outlines some of the conventions
+on development workflow, commit message formatting, contact points and
+other resources to make it easier to get your contributions accepted.
+
+## Project structure
+
+The project currently following structure:
+
+```
+.
+├── cmd
+│   └─<commands/sub-commands>.go
+├── pkg
+│   └── util
+│       ├── bpfutil
+│       │   └── all BPF command related files
+│       ├── dashboard
+│       │   └── all Dashboard command related files
+│       ├── diskutil
+│       │   └── all Disk command related files
+│       ├── netutils
+│       │   └── all Network command related files
+│       ├── procutil
+│       │   └── all Process command related files
+│       ├── sysutil
+│       │   └── all files related to generic System fucntionalities
+│       └── util.go ⟵ contains various utility functions that can be used by multiple packages
+└── Rest of files(README, Makefile, main.go, go.mod, go.sum, etc)
+```
+
+## Making A Change
+
+* _Before making any significant changes, please [open an
+issue](https://github.com/infracloudio/lot/issues)._ Discussing
+your proposed changes ahead of time will make the contribution process
+smooth for everyone.
+
+* Once we've discussed your changes and you've got your code ready,
+you can open your pull
+request against
+[`develop`](http://github.com/infracloudio/lot/tree/develop)
+branch.
+
```

Comment:
We should also add steps for building and testing project.

Reply:
We can't really add testing instructions as of now as its all manual now. Not a very good one, but the strategy we are following as of now basically is `it works if it compiles`

steps for building the project has already been added to [README](README.md)